src/osd/scrubber/osd_scrub.cc

```diff
@@ -294,47 +294,43 @@ std::optional<double> OsdScrub::LoadTracker::update_load_average()
 
   double loadavg;
   if (getloadavg(&loadavg, 1) == 1) {
```

Comment:
Not really related to this change, but `getloadavg` will work as expected only on containerized OSDs?

Reply:
Good question. I plan to check.